roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -118,6 +114,7 @@ def load_and_handle_cert(cert_string, now, base64decode=False):
 A 3-tuple of the form: (certificate_common_name, certificate_expiry_date, certificate_time_remaining)
 
     """
+    # pylint: disable=too-many-locals
```

Comment:
Indeed, basically, I've discovered that at some point we stopped running pylint against some of these files, so these disables were mostly to unblock test failures that were blocking further testing.